src/matrix.cu

```diff
@@ -327,6 +327,7 @@ Matrix< TemplateConfig<AMGX_device, t_vecPrec, t_matPrec, t_indPrec> >::print(ch
 
                     for (xdim = 0; xdim < this->get_block_dimx(); xdim++)
                     {
+			int loc = ii * this->get_block_dimx() * this->get_block_dimy() + this->get_block_dimy() * ydim + xdim;
```

Comment:
```suggestion
```

Reply:
Ah yes I was printing that for debugging output to see where I was! Will remove